src/error.rs

```diff
@@ -0,0 +1,8 @@
+/// A specialized [`Result`] type for turmoil simulations.
+///
+/// This type is generally useful for fallible test cases, i.e. where you want
+/// to use the `?` operator to fail the test rather than writing unwrap
+/// everywhere.
+///
+/// [`Result`]: std::result::Result
+pub type Result<T = ()> = std::result::Result<T, Box<dyn std::error::Error>>;
```

Comment:
You could consider depending on [`anyhow`](https://docs.rs/anyhow/latest/anyhow/). This is also fine though.

Reply:
Thanks for the suggestion. I'm going to stick with our own since it is a public API.

Reply:
I agree with our own type. However, I'd prefer `BoxResult` as the name.